activerecord/test/cases/associations/inverse_associations_test.rb

```diff
@@ -299,6 +299,17 @@ def test_parent_instance_should_be_shared_with_replaced_via_accessor_child
   def test_trying_to_use_inverses_that_dont_exist_should_raise_an_error
     assert_raise(ActiveRecord::InverseOfAssociationNotFoundError) { Man.first.dirty_face }
   end
+
+  if defined?(DidYouMean) && DidYouMean.respond_to?(:correct_error)
+    def test_trying_to_use_inverses_that_dont_exist_should_have_suggestions_for_fix
+      error = assert_raise(ActiveRecord::InverseOfAssociationNotFoundError) {
+        Man.first.dirty_face
```

Comment:
This was changed to `Human` in https://github.com/rails/rails/commit/7f938cacbac1128bde9b3b1399884d264203c970.

```suggestion
        Human.first.dirty_face
```

Reply:
That's a nice change. As I'm rereading this PR, maybe we should change the `horrible_man|face` as well.
https://github.com/rails/rails/blob/c93f3f02266cfbc81e802a8a7476844cfb73ba43/activerecord/test/models/face.rb#L10-L11

Reply:
Yeah, I found a bunch of the examples to be a bit on the unpleasant side (horrible, dirty, etc.). Depending on how large those changes are, it may make sense to do that in a separate PR.